src/data/markdown/docs/02 javascript api/09 k6-grpc.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "k6/grpc"
+excerpt: "k6 gRPC API"
+---
+
+The k6/grpc modules provides a [gRPC](https://grpc.io/) client for Remote Procedure Calls (RPC) over HTTP/2.
+
+| Function | Description |
+|----------|-------------|
+| [newClient()](/javascript-api/k6-grpc/newclient) | Creates a new [Client](/javascript-api/k6-grpc/client) that can be used to connect and make RPCs to a gRPC server. |
+
+| Class/Method | Description |
+|--------------|-------------|
+| [Client](/javascript-api/k6-grpc/client) | gRPC client used for making RPC calls to a gRPC Server |
+| [Client.load(importPaths, ...protoFiles)](/javascript-api/k6-grpc/client/client-load-importpaths----protofiles) | Loads and parses the given protocol buffer definitions to be made available for RPC requests. |
+| [Client.connect(address [,params])](/javascript-api/k6-grpc/client/client-connect-address-params) | Connects to a given gRPC service. |
+| [Client.invokeRPC(url, request [,params])](/javascript-api/k6-grpc/client/client-invokerpc-url-request-params) | Makes a unary RPC for the given service/method and returns a [Response](/javascript-api/k6-grpc/response). |
+| [Client.close()]() | Close the connection to the gRPC service. |
+| [Params](/javascript-api/k6-grpc/params) | RPC Request specific options. |
+| [Response](/javascript-api/k6-grpc/response) | Returned by RPC requests. |
+
+| Constant | Description |
+|----------|-------------|
+| `StatusOK` | OK is returned on success. |
```

Comment:
Do you have to repeat the name of the status on the second column? It looks redundant.